app/lib/job/job.dart

```diff
@@ -48,13 +50,16 @@ abstract class JobProcessor {
           _logger.info('$_serviceAsString job started: $jobDescription');
           sleepSeconds = 0;
 
-          JobStatus status;
+          JobStatus status = JobStatus.failed;
           try {
             status = await process(job);
-            status ??= JobStatus.success;
+            status = JobStatus.success;
             _logger.info('$_serviceAsString job completed: $jobDescription');
+          } on TransactionAbortedError catch (e, st) {
+            _logger.info('$_serviceAsString job error $jobDescription', e, st);
+          } on TimeoutError catch (e, st) {
+            _logger.info('$_serviceAsString job error $jobDescription', e, st);
```

Comment:
Maybe add a "(retrying later)" to the message if it is retried, so it's obvious it's harmless?

Reply:
I think it will be redundant, because it will be retried in all of the cases, regardless of the error type.